adr/data-test.md

```diff
@@ -0,0 +1,55 @@
+# [data-test] attributes
+
+## Context
+
+Quality Assurance teams perform automated tests on Products for non-regression testing.
+
+## Problems
+
+QA teams lean on HTML ID attributes or XPath to automate tests on UI, and updates on the shared library can break these scenarios.
+
+## Solutions
+
+Expose [data-test] attributes for QA purpose suggest in the components or layout markup.
+1 — By documenting them, the first benefit will be to act on a contract with our QA team.
+2 — By using them, after all, the second benefit will also be to perform component testing.
+
+```css
+[data-test="<block_name>.<element_type>[?<element_index>].<?element_identifier>"]
+```
+
+| Identifier           | Optional | Description                                                                                                                                                      |
+| -------------------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `block_name`         |          | Component or layout identifier from our design language (ex: modal, search, password, inlineediting, etc.)                                                       |
+| `element_type`       |          | Element or its type used (ex: button, link, input, textarea, radio, etc.)                                                                                        |
+| `element_index`      | yes      | Element index if it's repeated (radio, menu items, etc.)                                                                                                         |
+| `element_identifier` | yes      | A short and comprehensive identifier when we talk about it each other, in case of a form field we can reuse its label value (reveal, cancel, edit, submit, etc.) |
```

Comment:
```suggestion
| `element_identifier` | yes      | A short and comprehensive identifier. In case of a form field, we can use its label value (reveal, cancel, edit, submit, etc.) for instance. |
```